.github/actions/spell-check/patterns.txt

```diff
@@ -0,0 +1,5 @@
+(?:0[Xx]|U\+|#)[a-f0-9A-FGgRr]{2,}[Uu]?[Ll]{0,2}\b
+addNSECRecordToLW.*DNSName.*powerdnt.com.*QType::NS.*res->d_records
```

Comment:
This is because I don't want to whitelist `powerdnt` which is really a typo of `powerdns`.